locales/en.json

```diff
@@ -0,0 +1,11 @@
+{
+    "welcome": "your welcome to Phatom",
```

Comment:
There  is a simple typo with the welcome text, it should be "you are welcome to Phantom"

Reply:
Ohhhh!! okay

Reply:
Well solved